libc/cmake/modules/LLVMLibCHeaderRules.cmake

```diff
@@ -66,6 +67,90 @@ function(add_header target_name)
   )
 endfunction(add_header)
 
+function(add_yaml_header target_name)
```

Comment:
nit: This shouldn't be named `add_yaml_header` since the header itself doesn't have any yaml. Maybe `add_gen_header_2`?

Reply:
@michaelrj-google add_newgen_header?